src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -289,12 +287,72 @@ def __init__(
             "stream_cache_path": stream_cache_path,
         }
 
+    def read_nwb_file(self, file_path: str, stream_mode: Literal[None, "ffspec", "ros3"] = None, stream_cache_path: Optional[str] = None) -> NWBFile:
```

Comment:
Such a minor point I almost feel bad bringing it up... but we usually combine `nwb_file` into `nwbfile` in code (especially function and argument names)

Could this be `read_nwbfile` instead? 😅

Reply:
Yes, please do not feel bad, I appreciate that. Given that we are just now introducing this function. This is a good point.